bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrinoToDilepton_Schannel_NLO/HeavyNeutrinoToEE_Schannel_M1000_NLO/HeavyNeutrinoToEE_Schannel_M1000_NLO_run_card.dat

```diff
@@ -0,0 +1,153 @@
+#   mind the format:   value    = variable     ! comment               *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of events (and their normalization) and the required          *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+   100 = nevt_job! Max number of events per job in event generation. 
+                 !  (-1= no split).
+average = event_norm ! Normalize events to sum or average to the X sect.
+#***********************************************************************
+# Number of points per itegration channel (ignored for aMC@NLO runs)   *
+#***********************************************************************
+ 0.01   = req_acc_FO       ! Required accuracy (-1=ignored, and use the 
+ 	                   ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+ 5000   = npoints_FO_grid  ! number of points to setup grids
+ 4      = niters_FO_grid   ! number of iter. to setup grids
+ 10000  = npoints_FO       ! number of points to compute Xsec
+ 6      = niters_FO        ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+     0    = iseed       ! rnd seed (0=assigned automatically=default))
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+    1   = lpp1    ! beam 1 type (0 = no PDF)
+    1   = lpp2    ! beam 2 type (0 = no PDF)
+ 6500   = ebeam1  ! beam 1 energy in GeV
+ 6500   = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+ lhapdf    = pdlabel   ! PDF set                                     
+ 90200 = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
why not the standard 2017 PDF>

Reply:
Apart from this s-channel, we are also planning to request vbf t-channel process which looks like https://github.com/lesenka/GenLHEfiles/blob/master/GridpackWorkflow/production/exo_heavyNeutrino_VBF/templateCards/HeavyNeutrino_VBF_trilepton_template_NLO_proc_card.dat
This needs photon pdf so we are planning to use 82200 LUXqed17_plus_PDF4LHC15_nnlo_100 for this vbf t-channel process, it starts with "generate q a >".

This 90200 is the partner pdf of 82200 but without photons. So we thought it would be better to use corresponding pdfsets for the s-channel rather than default nnpdf which cms recommends.

Reply:
Hi Perrozzi, could you tell us what you think about our pdfset?

Reply:
I am not sure. I would naively still recommend to use the standard settings
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
you will get many different pdfs including those featuring the photon contribution, you can reweight at analysis level to the desired pdf